source/codegen/templates/service_helpers.mako

```diff
@@ -786,69 +790,69 @@ ${initialize_response_buffer(parameter_name=parameter_name, parameter=parameter)
 ${copy_to_response_with_transform(source_buffer=raw_response_field, parameter_name=parameter_name, transform_x=convert_x_to_enum, size=common_helpers.get_size_expression(parameter))}\
 %       elif parameter['type'] == 'ViReal64':
         if(${parameter_name} == (int)${parameter_name}) {
-          response->set_${parameter_name}(static_cast<${namespace_prefix}${parameter["enum"]}>(static_cast<int>(${parameter_name})));
```

Comment:
This method wasn't distinguishing "parameter_name" and "field_name", so I updated to make the "socket" in, "socket" out thing work in accept.